src/targets/gpu/CMakeLists.txt

```diff
@@ -131,8 +133,23 @@ add_library(migraphx_gpu
     time_op.cpp
     topk.cpp
     write_literals.cpp
-    ${JIT_GPU_SRCS}
+    jit/concat.cpp
+    jit/gather.cpp
+    jit/gathernd.cpp
+    jit/layernorm.cpp
+    jit/mlir.cpp
+    jit/pad.cpp
+    jit/pointwise.cpp
+    jit/reduce.cpp
+    jit/roialign.cpp
+    jit/scatternd.cpp
+    jit/softmax.cpp
```

Comment:
Use file globbing and just remove ck files from the list on windows.

Reply:
https://cmake.org/cmake/help/latest/command/file.html#glob

> **Note**: We do not recommend using GLOB to collect a list of source files from your source tree. If no `CMakeLists.txt` file changes when a source is added or removed then the generated build system cannot know when to ask CMake to regenerate. **The CONFIGURE_DEPENDS flag may not work reliably on all generators**, or if a new generator is added in the future that cannot support it, projects using it will be stuck. Even if CONFIGURE_DEPENDS works reliably, there is still a cost to perform the check on every rebuild.

Currently, among other reasons, because of `file(GLOB` in many places in CMakeLists.txt (not only MIGraphX), we face significant times to build a single project on Windows in Docker (>= 3h, e.g., for Debug versions). We want to shorten the time to build by reducing globbing as much as possible. I understand the list may change dynamically, and you don't want to keep updating it manually. If that is the case so, here is how we handled that in rocMLIR while building the FAT rockCompiler library: https://github.com/ROCmSoftwarePlatform/rocMLIR/pull/1160
If not, we prefer to keep it as it is.

Reply:
On the other hand, we will make it a separate PR in the future, so we will have the chance to discuss it more. I have changed the PR as you asked.